### PR TITLE
dbuild.sh: Refactor docker script

### PR DESCRIPTION
**Pull request by thapav**
1) Rename current docker options to:
======================================================
  "1. Build with Current Configuration"
  "2. Re-configure"
  "3. Modify Current Configuration"
  "4. Clean Build"
  "5. Clean Build and Re-Configure"
  "6. Build SmartFS Image"
  "d. Download"
  "t. Build Test"
  "x. Exit"
======================================================
2) Fix typo in the script excute > execute
3) Fix typo in the script treate > treat

Signed-off-by: Vidisha Thapa <thapa.v@samsung.com>

### CODE REVIEW

**Comment by sunghan-chang** [comment]
@thapav Regardless of this PR, how about renaming build content together?
```
			echo "  \"4. Build Clean\""  ---> Clean output
			echo "  \"5. Build Dist-Clean\"" --> Clean output + configuration
```
Because it's difficult to know difference between them.

**Comment by sunghan-chang** [comment]
```
echo " \"3. Menuconfig\"" ---> Adjust Configuration ??
```

**Comment by sunghan-chang** [comment]
@thapav There are several `treate`s. Could you fix them together?

**Comment by thapav** [comment]
> @thapav Regardless of this PR, how about renaming build content together?
> 
> ```
> 			echo "  \"4. Build Clean\""  ---> Clean output
> 			echo "  \"5. Build Dist-Clean\"" --> Clean output + configuration
> ```
> 
> Because it's difficult to know difference between them.

Yes, we could do this
> 			echo "  \"4. Build Clean\""  ---> Clean output
> 			echo "  \"5. Build Dist-Clean\"" --> Clean output + current configuration


> ```
> echo " \"3. Menuconfig\"" ---> Adjust Configuration ??
> ```

What about ---> Modify current configuration ??


> @thapav There are several `treate`s. Could you fix them together?

ok

**Comment by sunghan-chang** [comment]
> What about ---> Modify current configuration ??

How can we distinguish `2. reconfigure`? Is it easy to know the difference of them?
(I don't mean what I suggest could do.)

**Comment by thapav** [comment]
> > What about ---> Modify current configuration ??
> 
> How can we distinguish `2. reconfigure`? Is it easy to know the difference of them? (I don't mean what I suggest could do.)

How about ?
`
"1. Build with Current Configuration"
"2. Set up New Configuration"
"3. Modify Current Configuration"
"4. Clean Output"
"5. Clean Output + Current  Configuration"
`

**Comment by sunghan-chang** [comment]
@thapav Could you talk it with new guys who are unfamiliar to TizenRT?

File: os/dbuild.sh [hunk 1]
```diff
@@ -105,9 +105,9 @@ function SELECT_OPTION()
 			echo ======================================================
 			echo "  \"1. Build with Current Configurations\""
 			echo "  \"2. Re-configure\""
-			echo "  \"3. Menuconfig\""
-			echo "  \"4. Build Clean\""
-			echo "  \"5. Build Dist-Clean\""
+			echo "  \"3. Modify Current Configuration\""
+			echo "  \"4. Clean Build\""
+			echo "  \"5. Clean Build with New Configuration\""
```
Comment (abhishek-samsung) [inline os/dbuild.sh:110]: could you reword it to something else? Like **Clean Build and Clear Current Configuration** or something else? I feel it's better to inform the user that we are removing the current configuration.

Comment (sunghan-chang) [reply os/dbuild.sh:110]: I also think @abhishek-samsung 's suggestion is better to know what it is.
And minor one.
`configurations` or `configuration`? Could you use the same at all of the list?
`1. build  with current configurations` has `s`.

Comment (thapav) [reply os/dbuild.sh:110]: Yes Clean Build and Re-configuration is more precise.
I have modified all to configuration.

File: os/dbuild.sh [hunk 2]
```diff
@@ -103,11 +103,11 @@ function SELECT_OPTION()
...
 			echo "  \"Select build Option\""
 			echo ======================================================
-			echo "  \"1. Build with Current Configurations\""
+			echo "  \"1. Build with Current Configuration\""
 			echo "  \"2. Re-configure\""
-			echo "  \"3. Menuconfig\""
-			echo "  \"4. Build Clean\""
-			echo "  \"5. Build Dist-Clean\""
+			echo "  \"3. Modify Current Configuration\""
+			echo "  \"4. Clean Build\""
```
Comment (sunghan-chang) [inline os/dbuild.sh:109]: 1. How about this? `Clean build`? `Clean output`?
2. for 5, Re-Configuration -> Re-configure

Comment (thapav) [reply os/dbuild.sh:109]: 1. Lets keep it as `Clean build` 
2. `Re-configure` sounds good

Comment (sunghan-chang) [reply os/dbuild.sh:109]: ok

Comment (sunghan-chang) [reply os/dbuild.sh:109]: Could you update the commit description with new sentence?

Comment (thapav) [reply os/dbuild.sh:109]: done

**Review by sunghan-chang — APPROVED** [review]
